src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const keysValues = sourceString.split(';');
```

Comment:
The variable name 'keysValues' could be improved for better readability and context understanding. Consider a name that reflects the content it holds, such as 'styleDeclarations' or 'cssDeclarations'.